falcon/hooks.py

```diff
@@ -29,26 +29,29 @@
 def before(action, *args, **kwargs):
     """Decorator to execute the given action function *before* the responder.
 
+    The `params` argument that is passed to the hook
```

Comment:
While I was at it, I went ahead and added this clarification although it isn't strictly in keeping with the topic of this patch.